using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md

```diff
@@ -0,0 +1,178 @@
+| `triggeredconsequence`| Array |
+| `loadedconsequences` | Array |
+
+#### Event data example
+
+```text
+{ 
+    "loadedconsequences": [
+    {
+        "assetsPath":"assets/path/", 
+        "detail":"details", 
+        "id":"123", 
+        "type":"iam"
+    },
+    {
+        "assetsPath":"assets/path/", 
+        "detail":"details", 
+        "id":"456", 
+        "type":"csp"
+    }]
+}
+```
+
+
+### Configuration Response Content  <a id="configuration-response-content"></a>
+
+The data property in this event is used by each extension to modify its current settings. Each extension is responsible for reading out the part of the data property for which it is concerned.
+
+#### Data payload definition  <a id="data-payload-definition-3"></a>
+
+The Adobe Campaign Standard extension will read the following key from the configuration event:
+
+| **Key** | **Value Type** | **Optional** | **Description** |
+| :--- | :--- | :--- | :--- |
+| `__dev__campaign.server` | String | Yes | This contains the endpoint URL for the development environment Adobe Campaign Standard instance |
+| `__dev__campaign.pkey` | String | Yes | This contains the identifier for a mobile app configured in development environment Adobe Campaign Standard |
+| `__stage__campaign.server` | String | Yes | This contains the endpoint URL for the stage environment Adobe Campaign Standard instance |
+| `__stage__campaign.pkey` | String | Yes | This contains the identifier for a mobile app configured in stage environment Adobe Campaign Standard |
+| `campaign.server` | String | Yes | This contains the endpoint URL for the production environment Adobe Campaign Standard instance |
+| `campaign.pkey` | String | Yes | This contains the identifier for a mobile app configured in production environment Adobe Campaign Standard |
+| `campaign.mcias` | String | Yes | This contains the in-app messaging service URL endpoint |
+| `campaign.timeout` | String | Yes | This contains the amount of time to wait for a response from in-app messaging service |
+| `global.privacy` | Boolean | Yes | This contains the mobile privacy status settings |
+
+{% hint style="info" %}
+If `global.privacy` is changed to **optout**, linkage fields will be reset. All downloaded messages, rules will be erased and no tracking request may leave the device.
+{% endhint %}
+
+#### Event data example
+
+```text
+{
+    "global.privacy":"optedin",
+    "campaign.timeout":1,
+    "campaign.mcias":"mcias-va7.cloud.adobe.io/mcias",
+    "__dev__campaign.pkey":"",
+    "__stage__campaign.pkey":"",
+    "campaign.pkey":"",
+    "__dev__campaign.server":"mcias.campaign-demo.adobe.com",
+    "__stage__campaign.server":"mcias.campaign-demo.adobe.com",
+    "campaign.server":"mcias.campaign-demo.adobe.com"
+}
+```
+
+### Request Identity  <a id="request-identity"></a>
+
+This event is dispatched by public API to setLinkageFields to set the linkage fields. It is used to download personalized In-App messages.
```

Comment:
Please reword as follows:

"This event is dispatched by the API to setLinkageFields, which sets the linkage fields. It is used to download personalized in-app messages."